pkg/api/grpcServer_test.go

```diff
@@ -135,16 +142,16 @@ func (ts *testState) Quota(ctx context.Context, requestBag *attribute.MutableBag
 }
 
 func TestCheck(t *testing.T) {
-	ts, err := prepTestState(29999)
+	ts, err := prepTestState()
 	if err != nil {
-		t.Errorf("unable to prep test state %v", err)
+		t.Fatalf("Unable to prep test state: %v", err)
 		return
```

Comment:
so, if we use t.FatalF, we shouldn't need the `return`. It is supposed to stop the test execution immediately, which is why I originally suggested it.